### ISSUE TITLE
 Cognate suggestions: Failed to compute analisys

### ISSUE THREAD

**Issue by nadezhday** [ispras/lingvodoc-react#571]
We have a new DB

Test tests\test_cognate_suggestions.py (test_lex_cognate_sugg_unselect_match) - Failed
Steps to reproduce:

Login as admin
Open http://83.149.198.62/dictionary/1106/7053/perspective/1106/7054/view
Tools > Cognate_suggestions

Deselect Match translations

Compute

Expected: Analysis results

Actual: Exception

![image](https://user-images.githubusercontent.com/59951991/147092270-27e4668a-98d7-460a-9a48-87e41e42b2ae.png)

Error: GraphQL error: Exception:
Traceback (most recent call last):
  File "/api/lingvodoc/schema/query.py", line 6629, in mutate
    return CognateAnalysis.perform_cognate_analysis(
  File "/api/lingvodoc/schema/query.py", line 5417, in perform_cognate_analysis
    output_buffer = ctypes.create_unicode_buffer(output_buffer_size + 256)
  File "/usr/lib/python3.8/ctypes/__init__.py", line 295, in create_unicode_buffer
    buftype = c_wchar * init
ValueError: Array length must be >= 0, not -2035263040
    e ApolloError.js:37
    next QueryManager.js:186
    d Observable.js:126
    h Observable.js:161
    value Observable.js:215
    next index.js:37
    d Observable.js:126
    h Observable.js:161
    value Observable.js:215
    l index.js:91
    promise callback*l/</< index.js:90
    t Observable.js:179
    value Observable.js:258
    o index.js:19
    t Observable.js:179
    value Observable.js:258
    _ QueryManager.js:183
    mutate QueryManager.js:124
    mutate ApolloClient.js:98
    dataForChildViaMutation react-apollo.browser.umd.js:544
    value index.js:2172
    Lodash 4
    M Button.js:103
    React 12
    unstable_runWithPriority scheduler.production.min.js:19
    React 3
    d raven.js:393

GraphQL error: Exception: Traceback (most recent call last): File "/api/lingvodoc/schema/query.py", line 6629, in mutate return CognateAnalysis.perform_cognate_analysis( File "/api/lingvodoc/schema/query.py", line 5417, in perform_cognate_analysis output_buffer = ctypes.create_unicode_buffer(output_buffer_size + 256) File "/usr/lib/python3.8/ctypes/__init__.py", line 295, in create_unicode_buffer buftype = c_wchar * init ValueError: Array length must be >= 0, not -2035263040

**Comment by myrix** [comment]
That was an error in an external library https://github.com/ispras/lingvodoc-ext-oslon due to a too large data size. I sidestepped the error for now, but for real solution we would need to look to the library's author, Mikhail Oslon, who right now does not work with Lingvodoc, so here's hoping this error won't appear any time soon.

@nadezhday 
Even if it works, it still requires really large amounts of RAM, somewhere around 17 Gb as my local test have shown, due to large amount of data taken from a large amount of dictionaries in this sublanguage; I suggest, if it's possible, to move this test to a dictionary in another sublanguage, otherwise the test would be quite a strain on the test stand.

Edit: Would probably take a long time too. Tested on production, takes 9 Gb of RAM and several minutes to complete.
